include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_fpga.h

```diff
@@ -59,7 +59,9 @@ struct __parallel_for_fpga_submitter<__internal::__optional_kernel_name<_Name...
     __future<sycl::event>
     operator()(_ExecutionPolicy&& __exec, _Fp __brick, _Index __count, _Ranges&&... __rngs) const
     {
-        assert(oneapi::dpl::__ranges::__get_first_range_size(__rngs...) > 0);
+        using _Size = std::make_unsigned_t<std::common_type_t<oneapi::dpl::__internal::__difference_t<_Ranges>...>>;
+        assert(std::min({_Size(__rngs.size())...}) > 0);
+        assert(__count> 0);
```

Comment:
`_Size` is not used outside of asserts. That will result in a warning. 
> parallel_backend_sycl_fpga.h:62:15: warning: unused type alias '_Size' [-Wunused-local-typedef]

Reply:
Removed "using _Size" to avoid the warning.